src/web/components/HeadlineByline.tsx

```diff
@@ -152,8 +150,9 @@ export const HeadlineByline = ({
                     return (
                         <div
                             className={`${opinionStyles(pillar)} ${
-                                tags.find((tag) => tag.bylineImageUrl)
-                                    ? mobileCommentAuthor
+                                tags.filter((tag) => tag.type === 'Contributor')
+                                    .length === 1
+                                    ? authorBylineWithImage
                                     : ''
```

Comment:
I know this is merged but when we look at logic wider, it would be better to assign this to a meaningful class and use the cx helper:

`const hasSingleContributor = tags.filter((tag) => tag.type === 'Contributor').length === 1`

`cx({ [authorBylineWithImage]: hasSingleContributor }, opinionStyles(pillar))`

and then extend that with the other checks like: 

`cx({ [authorBylineWithImage]: hasSingleContributor && authorNameMatchesBlah && thisOtherThing}, opinionStyles(pillar))`